docs/spelling_wordlist

```diff
@@ -840,6 +840,8 @@ Tomek
 toolbar
 toolkits
 toolset
+tox
```

Comment:
I think you shouldn't need the lowercase version as long as all instances in the docs are surrounded by backticks.